### DISCUSSION TITLE
Welcome to LinuxGSM-Docker Discussions!

### DISCUSSION THREAD

**Discussion by dgibbs64** [GameServerManagers/LinuxGSM-Docker#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by takov751** [comment]
Greetings . I am just an enthusiast  whom just runs a vallheim server inside a docker(with this project) and have experience with docker/docker-compose (hosted and created a basic docker infra for a minecraft server with multiple alt-servers). I wanted to help out in this project as the linuxgsm is a great tool, but when it comes to moving and getting comfy on a new VPS. Installing the whole stack or bare system might cause some security issues( especially the game server is old and vulnerable). Which is why I think for a container around it would make it significantly easier to setup, backup and secure.
